types.ts

```diff
@@ -10,11 +10,20 @@ export interface INode {
     id?: string; //Could there be a more specific ID than just number?
     x: number;
     y: number;
+    project_id: number;
 }
 
 export interface IEdge {
     source_id: string;
     target_id: string;
+    project_id: number;
+}
+
+export interface IProject {
+    id: number;
+    owner_id: string;
+    name: string;
+    description: string;
 }
```

Comment:
We can debate whether any id should be a string or a number. Yes, the DB creates a serial number for the ids. However, all previous ids have already been changed to string. Having them as strings now would make it easier in the possible future to have id's be something more complex rather than just a number.

Reply:
Currently the id typing is broken throughout the whole app and this is the only one working correctly due to using numbers. ID refactor for the whole app will be done in a different feature branch